### ISSUE TITLE
setDirection and setSpeed need to run on componentDidMount

### ISSUE THREAD

**Issue by nlfonseca** [chenqingspring/react-lottie#22]
Hello,

`setDirection()` and `setSpeed()` runs only on `componentDidUpdate()` and needs to run also on `componentDidMount()` if we want that default direction is `-1` for example.

**Comment by urgentMilkMan** [comment]
YES THIS. Seriously this was driving me NUTS. I do it on the wrapper now. But it's insane this isn't supported.

**Comment by urgentMilkMan** [comment]
Here is an example of a simple fix around.

https://codesandbox.io/embed/powmy7llx
